opcache.php

```diff
@@ -3,7 +3,7 @@
  * Plugin Name: OPcache Dashboard
  * Plugin URI: http://wordpress.org/plugins/opcache/
  * Description: OPcache dashboard designed for WordPress
- * Version: 0.3.1
+ * Version: 0.3.2
```

Comment:
The version should be 0.3.2-dev or something. (because 0.3.2 is not released at this time.)